packages/ui/package.json

```diff
@@ -9,7 +9,7 @@
   "types": "./dist/index.d.ts",
   "license": "MIT",
   "scripts": {
-    "build": "tsup",
+    "build": "tsup --format cjs,esm",
```

Comment:
Supposedly, this can go into `tsup.'config.ts`.  Did that now work?

Reply:
Yea this was already in tsup config. I was experimenting but never cleaned it up. I'll push a cleanup for it